demo/Demo/Lists.elm

```diff
@@ -0,0 +1,307 @@
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , Options.span [] [ text "Bob Odenkirk" ]
+                ]
+            , Options.styled Html.a [ Lists.itemSecondaryAction ] [ Options.styled Html.i [] [ Icon.view "star" [] ] ]
+            ]
+        ]
+
+
+avatarsAndControls : Model -> Html Msg
+avatarsAndControls model =
+    Lists.ul [ Options.css "width" "300px" ]
+        [ Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , text "Bryan Cranston"
+                ]
+            , Options.span [ Lists.itemSecondaryAction ]
+                [ Toggles.checkbox Mdl
+                    [ 0 ]
+                    model.mdl
+                    [ Toggles.value True
+                    ]
+                    []
+                ]
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , text "Aaron Paul"
+                ]
+            , Options.span [ Lists.itemSecondaryAction ]
+                [ Toggles.radio Mdl
+                    [ 0 ]
+                    model.mdl
+                    [ Toggles.value False
+                    , Options.css "display" "inline"
+                    ]
+                    []
+                ]
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , text "Bob Odenkirk"
+                ]
+            , Options.span [ Lists.itemSecondaryAction ]
+                [ Toggles.switch Mdl
+                    [ 0 ]
+                    model.mdl
+                    [ Toggles.value True
+                    ]
+                    []
+                ]
+            ]
+        ]
+
+
+twoLines : Model -> Html Msg
+twoLines model =
+    Lists.ul [ Options.css "width" "300px" ]
+        [ Lists.li [ Lists.twoLine ]
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , Options.span [] [ text "Bryan Cranston" ]
+                , Options.span [ Lists.itemSubTitle ] [ text "62 Episodes" ]
+                ]
+            , Options.span [ Lists.itemSecondaryContent ]
+                [ Options.span [ Lists.itemSecondaryInfo ] [ text "Actor" ]
+                , Options.styled Html.a [ Lists.itemSecondaryAction ] [ Icon.view "star" [] ]
+                ]
+            ]
+        , Lists.li [ Lists.twoLine ]
+            [ Options.span [ Lists.itemPrimaryContent ]
```

Comment:
Are `itemSecondaryContent` and others always `span`, or could they conceivably sometimes be sometimes else, like `div`? If they are always `span`, maybe define them as such:

    itemSecondaryContent options = 
      Options.span (cs "mdl-list--secondary-content" ++ options)